devel_subthemer.module

```diff
@@ -141,7 +141,8 @@ function devel_subthemer_form($form, &$form_state, $theme = NULL) {
     '#description' => t('Enter relative paths, one per line. A blank JS file will be created at this path. For example <em>js/scripts.js</em> (file placed in a "js" folder) or <em>js-scripts.js</em> (file placed in root).'),
     '#rows' => 3,
   );
-  $form['convert'] = array(
+  $form['actions'] = array('#type' => 'actions');
+  $form['actions']['convert'] = array(
```

Comment:
We should probably take the opportunity to change from "convert"; thats a carryover from a Coder Upgrade conversion function.

Reply:
```suggestion
  $form['actions']['create_subtheme'] = array(
```